package.json

```diff
@@ -20,7 +20,7 @@
     "standard": "^8.0.0"
   },
   "engines": {
-    "node": "5.*"
+    "node": "<6"
```

Comment:
By setting `"<6"` on this line, you're allowing developers to use Node version `0.*`.

However, [Standard](http://standardjs.com/) cannot run on Node below version `4.0.0`:

![standard](https://cloud.githubusercontent.com/assets/183227/19413278/77bd9ac8-92dd-11e6-8c6e-c21dc01fbe18.png)